site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+  </tr>
+</table>
+
+The browser handles sign-up and sign-in use cases differently depending on the
+existence of `approved_clients` in the response from [the accounts list
+endpoint](#accounts-list-endpoint). The browser will only display [the RP's
+privacy policy and terms of service](#client-metadata-endpoint) in the dialog if
+`approved_clients` isn't provided or does not include the RP's `clientId` and
+the user hasn't previously signed up for the RP in this browser.
+
+<figure class="float-right screenshot" style="max-width:300px">
+{% Video
+   src="video/YLflGBAPWecgtKJLqCJHSzHqe2J2/Qx48SEGIEqi5OtPE9ogn.mp4",
+   width="280", autoplay="true"
+%}
+  <figcaption>A user signs into an RP using FedCM</figcaption>
+</figure>
 
-Read detailed instructions on how to implement and [participate in the
-third-party origin trial](/blog/fedcm-origin-trial) for FedCM for support details.
+When the RP calls `navigator.credentials.get()`, the following activities take
+place:
+
+1. The browser sends requests and fetches several documents:
+    1. [The well-known file](#well-known-file) and [an IdP config
+       file](#idp-config-file) which declare endpoints.
+    2. [An accounts list](#accounts-list-endpoint).
+    3. Optional: URLs for the RP's privacy policy and terms of service,
+       retrieved from the [client metadata endpoint](#client-metadata-endpoint).
+2. The browser displays the list of accounts that the user can use to sign-in,
+   as well as the terms of service and privacy policy if available.
+3. Once the user chooses an account to sign in with, a request to [the ID
+   assertion endpoint](#id-assertion-endpoint) is sent to the IdP to retrieve a
+   token.
+4. The RP can validate the token to authenticate the user.
+
+{%
+  Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/8EzrPvIwOsuuzdKJUJ6E.png",
+alt="login API call", width="800", height="419", class="type--full-bleed"
+%}
+
+{% Aside 'caution' %}
+
+FedCM is designed to not inform the RP of the user's IdP sign-in state until the
+user explicitly confirms to **Continue as** and signs in. This means RPs aren't
+informed of connection to the FedCM API if: the user isn't signed into the IdP,
+the accounts list endpoint returns an empty list, or the endpoint returns an
+error.
+
+RPs are expected to support browsers which don't support FedCM, therefore
+users should be able to use an existing, non-FedCM sign-in process. Until
+third-party cookies are phased out completely, this should remain
+non-problematic.
+
+{% endAside %}
+
+Once the token is validated by the RP server, the RP may register the user or
+let them sign-in and start a new session.
 
 ## Engage and share feedback {: #share-feedback}
 
-*  **Origin trial**: an [origin trial for
-   FedCM](/origintrials/#/view_trial/3977804370874990593) is available in Chrome from version 101 to 107. Learn more about [the origin trial](/blog/fedcm-origin-trial).
 *  **GitHub**: Read the
```

Comment:
Should probably link the explainer too as that is most relevant https://github.com/fedidcg/FedCM/blob/main/explainer.md